lib/rallybot.ex

```diff
@@ -63,7 +63,7 @@ defmodule Rallybot do
   end
 
   def handle_group_add(_, group_name, _, state) do
-    {:reply, %Rallybot.Response{text: "Group '#{group_name}' not found."}, state}
```

Comment:
I have no idea if those brackets need to be escaped.

Reply:
Nope! LGTM :+1: